index.html

```diff
@@ -89,5 +89,193 @@ <h1 id="header-text">
         </div>
       </section>
     </section>
+
+   <!-- the next project code starts here -->
+   <section class="works-section">
+     <div class="project-1" id="project-1">
+       <div id="tonic-picture">
+         <img src="./images/Snapshoot Portfolio.svg" alt="snap-shot" class="img-project"/>
+         <h2 class="tonic multi-post">Tonic</h2>
+          <div class="radio-btns canopy-1">
+           <label for="" class="canopy">CANOPY 
+             <input type="radio" name="" id="" value="" class="radio" /> </label>
+             <label for="" class="back-end">Back End Dev<input type="radio" name="" id="" value="" class="radio" /></label>
+           <label for="" class="back-end 2015">2015</label>
+          </div>
+            <p class="prjct-text">
+             A daily selection of privately personalized reads; no accounts or
+             sign-ups required
+            </p>
+          <div class="lang-btn lang-btn-ul">
+           <ul class="lang-btn-ul">
+             <li><button class="btn" >HTML</button></li>
+             <li><button class="btn" >CSS</button></li>
+             <li><button class="btn" >JavaScript</button></li>
+            </ul>          
+          </div>
+          <button class="button btn-border"><a href="#" class="button">See Project</a></button>
+      </div>
```

Comment:
- [ ] Let's focus on this card as the case study. I am not able to access some of the stylings since there was an inheritance of colors from other sections.  Kindly stick to the Figma design as much as possible. This is particularly on the colors. The colors and border colors mostly used on this card, include `#172B4D`, `#344563`, `#7A869A`. Please modify the colors to match the Figma design. This will make our project more professional.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/228257732-2755584c-d328-4662-a58d-5e243ee02a67.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/228258239-512b7079-d47f-46aa-8af9-8a3ef9316bcd.png)